src/intersections.jl

```diff
@@ -44,6 +44,9 @@ The different types of intersection that may occur between geometries.
 Type `IntersectionType` in a Julia session to see the full list.
 """
 @enum IntersectionType begin
+  # no intersection
+  NoIntersection
```

Comment:
`NoIntersection` is now linked to zero

Reply:
I was already linked to zero in the latest master version. Please start your PR based on the latest master branch.

Reply:
I tried to undo all my local changes and start from the original master file. Hope it worked.